src/icon-buttons/icon-button.css

```diff
@@ -0,0 +1,12 @@
+.bandicootButton {
+  background-color: transparent;
+  border: none;
+  border-radius: 4px;
+}
+.bandicootButton:hover {
+  transition: background-color .25s ease-in-out;
+  background-color: rgba(0, 0, 0, 0.05);
+}
```

Comment:
should we even have these default styles? Maybe we don't change button or its hover and all we do is tell the svg to use the color of the parent. Then perhaps we don't even need postcss at all because adding one style to the svg is enough